common/src/main/java/com/revenuecat/purchases/common/diagnostics/DiagnosticsEntry.kt

```diff
@@ -43,7 +43,7 @@ sealed class DiagnosticsEvent(val diagnosticType: String) {
         val name: String,
         val tags: List<String>,
         val value: Int
-    ) : DiagnosticsEvent("metric") {
+    ) : DiagnosticsEntry("metric") {
```

Comment:
I'm still confirming whether we need to send any entries like this. But if not, we can probably just remove `DiagnosticsEntry` entirely and just leave `DiagnosticsEvent` as a data class.